sklearn/linear_model/sag_fast.pyx

```diff
@@ -614,10 +614,14 @@ cdef void lagged_update(double* weights, double wscale, int xnnz,
                         last_update_ind = sample_itr - 1
                     for lagged_ind in range(sample_itr - 1,
                                    last_update_ind - 1, -1):
-                        grad_step = (cumulative_sums[lagged_ind]
-                           - cumulative_sums[lagged_ind - 1])
-                        prox_step = (cumulative_sums_prox[lagged_ind]
-                           - cumulative_sums_prox[lagged_ind - 1])
+                        if lagged_ind > 1:
```

Comment:
The wraparound, etc, flags do not apply at all to pointers.

But the basic issue is that O/Ses allocate memory on the heap differently. Most of it tends to be zero (if only because we use 64 bit int arrays to store small numbers). So this just worked when the array happened to be allocated next to a zero ... which always happened on Travis and AppVeyor.